ovh/client.py

```diff
@@ -479,12 +479,17 @@ def raw_call(self, method, path, data=None, need_auth=True):
         :param str path: api entrypoint to call, relative to endpoint base path
         :param data: any json serializable data to send as request's body
         :param boolean need_auth: if False, bypass signature
+        :param dict headers: A dict containing the headers needing to be sent to
+                             the OVH API. The function may override the
+                             OVH API authentication headers, as well as
+                             the Content-Type header.
         """
         body = ''
         target = self._endpoint + path
-        headers = {
-            'X-Ovh-Application': self._application_key
-        }
+    
+        if headers is None:
```

Comment:
Can't we have `headers={}` as a default instead?

Reply:
I consider bad practice to have default arguments being mutable, so i'd rather have None and handle the condition in the code

Reply:
Indeed, I didn't know Python's behaviour regarding defaults... Better stay away from them :smile: